MacPass/MPPasswordInputController.m

```diff
@@ -81,24 +91,27 @@ - (void)viewDidLoad {
   [self.enablePasswordCheckBox bind:NSValueBinding toObject:self withKeyPath:NSStringFromSelector(@selector(enablePassword)) options:nil];
   [self.togglePasswordButton bind:NSEnabledBinding toObject:self withKeyPath:NSStringFromSelector(@selector(enablePassword)) options:nil];
   [self.passwordTextField bind:NSEnabledBinding toObject:self withKeyPath:NSStringFromSelector(@selector(enablePassword)) options:nil];
+  self.touchIdEnabled.hidden = true;
+  if (@available(macOS 10.13.4, *)) {
+    self.touchIdEnabled.hidden = false;
+    self.touchIdEnabled.state = [NSUserDefaults.standardUserDefaults integerForKey:kMPSettingsKeyEntryTouchIdEnabled];
```

Comment:
It might be usefull to bind an internal variable to ```[NSUserDefaults.standardUserDefaults integerForKey:kMPSettingsKeyEntryTouchIdEnabled]``` and bind UI to this internal variable to ensure better update flow. This is an edge case which might not make the trouble of binding worth it.